lib/matplotlib/contour.py

```diff
@@ -406,97 +406,6 @@ def interp_vec(x, xp, fp): return [np.interp(x, xp, col) for col in fp.T]
 
         return angle, Path(xys, codes)
 
-    @_api.deprecated("3.8")
-    def calc_label_rot_and_inline(self, slc, ind, lw, lc=None, spacing=5):
```

Comment:
Not documented?

Reply:
I couldn't see a changenote for these two at v3.8 so followed that.  My impression is that, despite being technically public, these would only have been used internally.

Reply:
That's odd; I'd still document that it was removed though.